app/views/layouts/application.html.erb

```diff
@@ -12,7 +12,11 @@
   <body>
     <header>
       <nav id="nav">
-      <%= link_to image_tag('logo.png'), root_path %>
+        <form class="form-inline">
+          <%= link_to image_tag('logo.png'), root_path %>
+          <%= link_to 'Admin', admin_root_path, class: "btn btn-outline-dark" %>
```

Comment:
user won't have access to admin panel. why would we add this link?

Reply:
Remove link to admin
Addressed here https://github.com/vova/war_shelter/pull/86/commits/c83c700bc97276e7d3ad109467d4c6a6688c2502